src/flyout.js

```diff
@@ -51,7 +51,9 @@ export default class Flyout extends React.Component {
       style.display = 'none';
     }
 
-    return <div {...this.props} style={style}>{this.props.children}</div>;
+    const { onRequestClose, ...divProps } = this.props
```

Comment:
- Use `let` instead of `const` (project's policy)
- Put a semicolon at the end of the line
- Object Rest/Spread Properties is still proposal. This project use only es2015 so please rewrite by another syntax.
   - https://github.com/tc39/proposal-object-rest-spread